### ISSUE TITLE
Welcome dialog: accessibility: left and right arrow buttons

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#8796]
### Bug description

These buttons are cannot be activated by keyboard users, and have no visual indication of focus.
For screen reader users, their role should be button, but it isn't.
The button used for these buttons is a custom button which doesn't  have the necessary functionality for accessibility. (wxButtons, AButtons, and GradientButtons all have the necessary functionality.)
I don't know why a completely new custom button was used.

### Steps to reproduce

With a windows screen reader running:
1. When the Welcome dialog opens, the left arrow button is the focus (although there is no visible indication of this).
2. Press Enter or Spacebar to press the button.

### Expected behavior

1. The screen reader reads that the focused item is a button.
2. Pressing Enter or Spacebar activates the button, and the carousel is moved back one slide.

### Actual behavior

1. The screen reader reads that the focused item is panel. (It also has no accessibility name, but I'll log that as a separate issue.
2. Pressing Spacebar does nothing, and pressing Enter closes the dialog.

### Audacity Version

Audacity3

### Operating system

Windows

**Comment by chinakov** [comment]
This is done, thanks!
